common/assets/scss/overrides/_positioning.scss

```diff
@@ -5,3 +5,15 @@
 .app-\!-float-left {
   float: left !important;
 }
+
+.app-\!-position-relative {
+  position: relative !important;
+}
+
+@media (min-width: 40.0625em) {
```

Comment:
Maybe use the media mixins that are provided as part of the GOV.UK Design System for consistency?